src/index.tsx

```diff
@@ -29,10 +29,12 @@ const router = createBrowserRouter(routes)
 
 const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement)
 
+const wantedTheme = router.state.location.pathname.startsWith('/testnet') ? testnetTheme : defaultTheme
```

Comment:
This should be moved into a component, and use useParams

Reply:
We is kind of a special case, since we are outside the router (and any other react components), and so useParams is not yet available.

But in any case, we have some dedicated components like that, in upcoming PRs.